main.go

```diff
@@ -37,14 +37,18 @@ func main() {
 			log.Fatal(err)
 		} else {
 			os.Setenv("REGISTRY_STORAGE_S3_ACCESSKEY", string(accesskey))
```

Comment:
@bacongobbler yeah, it's working for us. I think this change is required only for use by boto 2.